frontend/src/app/features/in-app-notifications/entry/in-app-notification-entry.component.html

```diff
@@ -1,118 +1,109 @@
-<div
-  class="op-ian-item"
-  [class.op-ian-item_expanded]="notification.expanded"
-  [class.op-ian-item_read]="notification.readIAN === true"
-  attr.data-qa-selector="op-ian-notification-item-{{notification.id}}"
-  [attr.data-qa-ian-read]="notification.readIAN === true || undefined"
+<button
+  type="button"
+  class="op-ian-item--row"
+  [class.op-ian-item--row_unexpandable]="unexpandable"
+  (click)="showDetails()"
 >
-  <button
-    type="button"
-    class="op-ian-item--row"
-    [class.op-ian-item--row_unexpandable]="unexpandable"
-    (click)="showDetails()"
-  >
-    <ng-container *ngIf="workPackage$ && (workPackage$ | async) as workPackage; else workPackageLoading">
+  <ng-container *ngIf="workPackage$ && (workPackage$ | async) as workPackage; else workPackageLoading">
+    <div
+      class="op-ian-item--top-line"
+    >
+      <div class="op-ian-item--status">
+        <op-in-app-notification-status [status]="workPackage.status"></op-in-app-notification-status>
+      </div>
+
+      <a
+        class="op-ian-item--work-package-id-link"
+        [attr.title]="workPackage.subject"
+        uiSref="work-packages.show"
+        [uiParams]="{workPackageId: workPackage.id}"
+        [textContent]="'#' + workPackage.id"
+      >
+      </a>
+
       <div
-        class="op-ian-item--top-line"
+        *ngIf="project"
+        class="op-ian-item--project"
       >
-        <div class="op-ian-item--status">
-          <op-in-app-notification-status [status]="workPackage.status"></op-in-app-notification-status>
-        </div>
+        - <a
+        class="op-ian-item--project-link" [href]="project.showUrl"
+        [textContent]="project.title"
+        (click)="projectClicked($event)"
+        target="_blank"
```

Comment:
This should probably be a button. If you have some time, maybe making the outer element a normal div instead of a button and doing the inner ones correctly could be worth it.

Reply:
I don't think that this should be a button. It is a classical link, it has no action on data, but just redirects to the overview page of that project.
I'll however remove the outer button and show a focusable element for opening the split screen as we discussed yesterday.